src/app/role-admin/employee-list/employee-list.component.css

```diff
@@ -0,0 +1,387 @@
+/* 
+*{
+    box-sizing: border-box;
+    overflow: hidden;
+} */
+
+
+.whole {
```

Comment:
have a meaningful name. it helps to understand the for what you are using the class